### PR TITLE
Cu24 keyboard addition

### PR DESCRIPTION
**Pull request by yiancar**
Addition of a new keyboard. Announcement can be found [here](https://geekhack.org/index.php?topic=93289.0) and it will be sold by [CapUnlocked](http://caps-unlocked.com/).

Production is expected at the end of the month!

### CODE REVIEW

**Comment by jackhumbert** [comment]
Awesome! Would you mind making the folder/filenames lowercase? We try to keep all the keyboards standardised like that :)

**Comment by yiancar** [comment]
Done!

File: .gitignore [hunk 1]
```diff
@@ -65,3 +65,4 @@ util/Win_Check_Output.txt
 secrets.tar
 id_rsa_*
 /.vs
+lib/chibios-contrib
```
Comment (jackhumbert) [inline .gitignore]: Were you having trouble with this? Our `make git-submodule` should be able to fix it. We need to be able to track changes to that folder :)

Comment (yiancar) [reply .gitignore]: submodules are synced but there is nothing new to git add so i cannot recommit :S

Comment (jackhumbert) [reply .gitignore]: Cool! Can you remove this line then? Just meant tracking changes in general, not with this specific PR :)

Comment (yiancar) [reply .gitignore]: Sorry this is embarrassing. I just realized what the issue was:)

Comment (yiancar) [reply .gitignore]: I think this should be ok now:)

**Comment by jackhumbert** [comment]
Perfect :) thanks!

**Pull request merged**